ui/lib/utils/validation.ts

```
@@ -1,153 +1,153 @@
+    }
+  }
+
+  static pattern(value: string, regex: RegExp, message: string): ValidationRule {
+    return {
+      isValid: regex.test(value || ''),
+      message,
+    }
+  }
+
+  static email(value: string, message = 'Must be a valid email'): ValidationRule {
+    return this.pattern(value, /^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,}$/i, message)
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)